clang/lib/CodeGen/CoverageMappingGen.cpp

```diff
@@ -1251,6 +1264,70 @@ struct CounterCoverageMappingBuilder
     popRegions(Index);
   }
 
+  /// Find a valid range starting with \p StartingLoc and ending before \p
+  /// BeforeLoc.
+  std::optional<SourceRange> findAreaStartingFromTo(SourceLocation StartingLoc,
```

Comment:
this is modified function `findGapAreaBetween` but we need to start at StartingLoc, not after